Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/MenuManager.cpp

```diff
@@ -185,5 +206,60 @@ namespace AzToolsFramework
 
         return menuBarIterator->second.GetMenuBar();
     }
+    
+    MenuManagerIntegerResult MenuManager::GetSortKeyOfActionInMenu(const AZStd::string& menuIdentifier, const AZStd::string& actionIdentifier)
```

Comment:
suggestion: This looks like it could be `const`?

Reply:
Fixed